crrExistingObjects.js

```diff
@@ -13,6 +13,10 @@ const ACCESS_KEY = process.env.ACCESS_KEY;
 const SECRET_KEY = process.env.SECRET_KEY;
 const ENDPOINT = process.env.ENDPOINT;
 const SITE_NAME = process.env.SITE_NAME;
+const PROCESS_ALL = ['1', 'true', 'yes', 'on'].includes(
```

Comment:
I think restricting the input is important, otherwise each person can have a different value used in their scripts. It will help validation as well. For example, if `-e "PROCESS_ALL=true"`, all you have to check for is
`const PROCESS_ALL = process.env.PROCESS_ALL === 'true;` You can throw an assertion if the user enters anything else.